src/control/server/storage/bdev/runner_test.go

```diff
@@ -109,7 +109,8 @@ func TestBdevRunnerPrepare(t *testing.T) {
 				DisableVFIO:   true,
 			},
 			expErr: errors.New(
-				"SPDK prepare: bdev_include and bdev_exclude can't be used together"),
+				"bdev prepare: re-binding ssds to attach with spdk: " +
```

Comment:
This is OK, but it would be better to just include the base string for your error ("bdev_include and bdev_exclude can't be used together"). The `common.CmpErr()` test helper uses `strings.Contains()` to verify that the expected error's message is found in the actual error's message. Doing it this way makes the tests less brittle as code changes.

Reply:
Done. Thanks for letting me know.